apps/pie-docs/src/foundations/motion/applying-motion/easing.md

```diff
@@ -0,0 +1,52 @@
+---
+eleventyNavigation:
+    key: Easing
+    parent: Applying Motion
+    order: 2
+---
+## Overview
+
+Easing controls the acceleration and deceleration of elements in motion. When paired with timing it allows us to create realistic and natural movements that mimic real-world physics and feel smooth and organic to the user.
+
+Easing can be categorised by three different types of movement; Entrance, Exit and Persistent. Entrance and Exit relate to the object entering and exiting from view, whilst Persistent movement remains in view. We use functional and expressive persistent easing to differentiate between functional and expressive motion.
+
+Motion that does not have easing applied is described as linear. Meaning that it goes from A to B without any acceleration or deceleration. This can feel unnatural to users.
+
+---
+
+## Entrance Easing (easing-out)
```

Comment:
Should entrance easing be "easing-in" and exit easing be "easing-out"?

Reply:
need to check with the designers

Reply:
Confirmed with design that this is correct, if initially a little confusing. The [MDN docs on easing functions](https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function#cubic-bezier-easing-function) helped me understand.